routes/configOverview.js

```diff
@@ -1,62 +1,24 @@
-
-var linz = require('../'),
-    async = require('async');
+var linz = require('../');
 
 /* GET /admin/config/:config/overview */
 var route = function (req, res) {
 
-	var locals = {
-		config: req.linz.config,
-		record: req.linz.record,
-		permissions: req.linz.config.linz.formtools.permissions,
-		formtools: req.linz.config.linz.formtools
-	};
-
-	async.series([
-
-        function (cb) {
-
-            req.linz.config.linz.formtools.overview.summary.renderer(req.linz.record, req.linz.config, function (err, content) {
-
-                if (err) {
-                    return cb(err);
-                }
-
-                locals.overviewSummary = content;
-
-                return cb(null);
-
-            });
-
-        },
-
-		function (cb) {
-
-			if (!req.linz.config.linz.formtools.overview.body) {
-
-				return cb(null);
-
-			}
-
-			req.linz.config.linz.formtools.overview.body(req.linz.record, req.linz.config, function (err, content) {
-
-                if (err) {
-                    return cb(err);
-                }
-
-				locals.overviewBody = content;
-
-				return cb(null);
-
-			});
+    var locals = {
+        config: req.linz.config,
+        record: req.linz.record,
```

Comment:
In `routes/recordOverview.js` `L12` it does a clone of the record, see code below:

```clone(req.linz.record.toObject({ virtuals: true}))```

Perhaps config should follow suit, in record do get manipulated downstream.